lib/OpenQA/Schema/Result/JobGroups.pm

```diff
@@ -107,4 +107,14 @@ around 'default_priority' => sub {
     return $self->get_column('default_priority') // ($self->parent ? $self->parent->default_priority : OpenQA::Schema::JobGroupDefaults::PRIORITY);
 };
 
+sub rendered_description {
+    my ($self) = @_;
+
+    if ($self->description) {
+        my $m = CommentsMarkdownParser->new;
+        return Mojo::ByteStream->new($m->markdown($self->description));
+    }
+    return 0;
```

Comment:
I know - that's why my first version (without any return) worked.

> (especially taken that you're not interested in numbers, but strings)

I know - so I tried `return undef` in the first place but this seems to be considered bad style. Actually I would like that because it makes the reason of this seemingly unnecessary `return` statement more clear without involving numbers.